frontend/src/axiosRequests.js

```diff
@@ -22,9 +22,8 @@ export const findPlace = async (searchedPlace, lat, lon) => {
   }
   return [];
 };
-
-export const getAttributesData = async (searchParams) => {
-  const response = await axios.get(`/api/attributes?${searchParams}`);
+export const getAttributesData = async (searchParams, baseUrl) => {
+  const response = await axios.get(`${baseUrl || '/api/attributes?'}${searchParams}`);
```

Comment:
Hi, could we use '/api/attributes?' as default param for baseUrl? And remove it from LN26 and LN48.